### PR TITLE
Update advanced-hunting-shared-queries.md

### PR DESCRIPTION
**Pull request by msbemba**
Tested and found only Global admin and Security admin can run this query. whereas the Security reader cannot run or share the query. Please validate and approve.

Thanks
Sri

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3857

File: microsoft-365/security/mtp/advanced-hunting-shared-queries.md [hunk 1]
```diff
@@ -35,6 +35,19 @@ ms.topic: article
 
 ![Image of shared queries](../../media/advanced-hunting-shared-queries.png)
 
+### Before you begin
+
+Users with one of the following levels of permissions to access Microsoft denfender.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/advanced-hunting-shared-queries.md]:
```suggestion
Users need one of the following levels of permissions to access Microsoft Defender:
```

File: microsoft-365/security/mtp/advanced-hunting-shared-queries.md [hunk 2]
```diff
@@ -35,6 +35,19 @@ ms.topic: article
...
+
+Full access (Read and Write)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/advanced-hunting-shared-queries.md]:
```suggestion
- Full access (read and write)
```

File: microsoft-365/security/mtp/advanced-hunting-shared-queries.md [hunk 3]
```diff
@@ -35,6 +35,19 @@ ms.topic: article
...
+Read-only access
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/advanced-hunting-shared-queries.md]:
```suggestion
- Read-only access
```

File: microsoft-365/security/mtp/advanced-hunting-shared-queries.md [hunk 4]
```diff
@@ -35,6 +35,19 @@ ms.topic: article
...
+
+**Full access**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/advanced-hunting-shared-queries.md]:
```suggestion
**Full access**:
```

File: microsoft-365/security/mtp/advanced-hunting-shared-queries.md [hunk 5]
```diff
@@ -35,6 +35,19 @@ ms.topic: article
...
+Users with full access can Save,Modify and share a query. Assigning full access rights requires adding the users to the "Security Administrator" or "Global Administrator" AAD built-in roles.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/advanced-hunting-shared-queries.md]:
```suggestion
Users with full access can save, modify, and share a query. Assigning full access rights requires adding the users to the "Security Administrator" or "Global Administrator" AAD built-in roles.
```

File: microsoft-365/security/mtp/advanced-hunting-shared-queries.md [hunk 6]
```diff
@@ -35,6 +35,19 @@ ms.topic: article
...
+
+**Read-only access**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/advanced-hunting-shared-queries.md]:
```suggestion
**Read-only access**:
```

File: microsoft-365/security/mtp/advanced-hunting-shared-queries.md [hunk 7]
```diff
@@ -35,6 +35,19 @@ ms.topic: article
...
+Users with read-only access can log in, view all alerts, and related information. They will not be able to save, modify or share a query. Assigning read-only access rights requires adding the users to the "Security Reader" Azure AD built-in role.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/advanced-hunting-shared-queries.md]:
```suggestion
Users with read-only access can log in and view all alerts and related information. They will not be able to save, modify, or share a query. Assigning read-only access rights requires adding the users to the "Security Reader" Azure AD built-in role.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lomayor Please merge this PR. Thanks.

**Comment by lomayor** [comment]
We should link to the permissions section of the AH overview topic instead of describing required permissions in this topic.

https://docs.microsoft.com/en-us/microsoft-365/security/mtp/advanced-hunting-overview?view=o365-worldwide#get-access

@schmurky -- fyi.

**Comment by msbemba** [comment]
@lomayor I'm closing this PR to avoid confusion. New PR# https://github.com/MicrosoftDocs/microsoft-365-docs/pull/4360

**Pull request closed without merging**